docs/encyclopedia/nexus-operations.mdx

```diff
@@ -0,0 +1,184 @@
+---
+id: nexus-operations
+title: Nexus Operations
+sidebar_label: Nexus Operations
+description: Discover how Nexus Service and Operations enable the creation of well-defined microservice contracts for sharing capabilities across team and Namespace boundaries, leveraging Temporal's durable and scalable infrastructure.
+toc_max_heading_level: 4
+slug: /nexus/operations
+keywords:
+  - nexus operations
+  - nexus lifecycle
+  - nexus operation lifecycle
+  - durable execution
+  - automatic retries
+  - execution semantics
+  - temporal sdk
+  - versioning
+  - nexus
+
+tags:
+  - Nexus
+  - Concepts
+
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/nexus).
+
+:::
+
+## Arbitrary-duration operations
+
+[Nexus Operations](/glossary#nexus-operation) are arbitrary-duration operations that may be synchronous or asynchronous, short-lived or long-lived, and are used to connect Temporal Applications within and across Namespaces, clusters, regions, and clouds.
+
+Unlike a traditional RPC, an asynchronous Nexus Operation has an operation identity that can be used to re-attach to a long-lived Nexus Operation, for example, one backed by a Temporal Workflow.
+Nexus Operations support a uniform interface to get the status of an operation or its result, receive a completion callback, or cancel the operation – all of which are fully integrated into the Temporal Platform.
+
+## SDK support
+
+The Temporal SDK provides an integrated Temporal experience to build, run, and use Nexus Operations.
+
+:::tip RESOURCES
+- [Go SDK - Nexus quick start and code sample](/develop/go/nexus)
+- [Java SDK - Nexus quick start and code sample](/develop/java/nexus)
+:::
+
+Caller Workflows use the Temporal SDK to execute Nexus Operations.
+
+Nexus Operation handlers are created with Temporal SDK builder functions such as:
+
+- New-Workflow-Run-Operation
+  - Start a Workflow as an asynchronous Operation.
+- New-Sync-Operation
+  - Invoke an underlying Query or Signal as a synchronous Operation.
+  - Invoke an Update as a synchronous Operation.
+  - Execute arbitrary code as a synchronous Operation.
+
+Nexus Operations are not intrinsically durable, but can be made durable by backing them with Temporal primitives like a Workflow, Update, or Signal.
```

Comment:
I find this statement confusing. i understand why you wrote this but I would clarify that on the caller side the operation is made durable and guaranteed to run to completion or timeout. This statement might scare people away from trying Nexus out.

Reply:
fair point, removed.